src/components/options/special/queries.svelte

```diff
@@ -168,16 +190,21 @@ onMount(async () => {
               type="text"
               bind:value="{item.query}"
               class="form-control text-info"
-              class:is-invalid="{hasErrors(index, 'query')}"
+              class:is-invalid="{hasError(index, 'query')}"
               placeholder="Query to be used on the requests"
               name="query_{index}"
               on:input="{(e) =>
                 validateInput(index, 'query', e.target.value, true)}"
               on:blur="{() => validateInput(index, 'query', item.query)}"
               on:change="{onQueryInputChanged}" />
-            {#if hasErrors(index, "query")}
+            {#if hasError(index, "query")}
               <div class="invalid-feedback ml-1">
-                {getErrors(index, "query")}
+                {getError(index, "query")}
+              </div>
+            {/if}
+            {#if hasWarning(index)}
+              <div class="text-warning text-small ml-1 mt-1">
+                {getWarning(index)}
```

Comment:
![image](https://user-images.githubusercontent.com/8558447/157507229-b1332a26-7a8b-48e8-a3a0-62860ef13fd1.png)

The `Query musn't be empty` error isn't longer being shown.

Also the warning says "link" instead of "query".

Reply:
Funny thing: I closed the tab, and re-open it and now it shows the empty error...

![image](https://user-images.githubusercontent.com/8558447/157511747-4e80f79c-4546-49a8-9806-e5cc4cd16834.png)

Anyway, we should show the warning only if there are no errors.